packages/arb-token-bridge-ui/package.json

```diff
@@ -5,7 +5,7 @@
   "private": true,
   "dependencies": {
     "@apollo/client": "^3.7.11",
-    "@arbitrum/sdk": "^3.3.3",
+    "@arbitrum/sdk": "npm:godzillaba-arbitrum-sdk",
```

Comment:
We need to change this to arbitrum/sdk when it gets merged. Our E2E tests are probably not running because of this.